deform/widget.py

```diff
@@ -1722,7 +1759,7 @@ class FileUploadWidget(Widget):
     readonly_template = "readonly/file_upload"
     accept = None
 
-    requirements = (("fileupload", None),)
+    requirements = ({"js": "deform:static/scripts/file_upload.js"},)
```

Comment:
I changed the requirements ere to the new form, as I feel this widget is tightly coupled to this bit of js. This is contradicting the statement 
> The two-tuple form is used by all core Deform widgets and uses the resource registry.

mentioned below

Reply:
Thanks! Fixed in 8d25cc9.